docker/docker-compose.dev.yml

```diff
@@ -12,7 +12,22 @@ services:
     volumes:
       - ..:/srv/app
     command: "flask run --host=0.0.0.0"
+    networks:
```

Comment:
what's the purpose of a custom network? Containers in docker-compose join the default network automatically and already see each other using the service name as the hostname (`elasticsearch` here)

Reply:
This was left over from my experimentation using the full ELK stack. I'll see if I can remove it.

Reply:
Removed.